CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ## Unreleased
 
 * Drop support for Ruby 3.1 ([PR #4124](https://github.com/alphagov/govuk_publishing_components/pull/4124))
+* **BREAKING:** Update cards component design and use one column by default ([PR #4118](https://github.com/alphagov/govuk_publishing_components/pull/4118))
```

Comment:
Is this actually a breaking change? What's the worst that could happen if it was rolled out without changes in an application?

Reply:
Yes I know what you mean, it does feel quite a lot to mark this as a breaking change, but this does seem the correct approach following [semver](https://semver.org/): 

> MAJOR version when you make incompatible API changes

The worst that would happen in this case is that if you are using the card component with a default 3 column layout, this will now change to be a single column layout and you will need to follow the guidance to use a 3 column layout again if required.

Reply:
I'd agree with semver, because it's sensible, although in this case it doesn't feel _that_ bad as a thing that would happen? But I'm not going to argue this.